### DISCUSSION TITLE
Suggestion: Change "CUDA Settings" to something more encompassing.

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#813] [Ideas]
Since CUDA Settings is now a place for all Compute method options (DirectML, ROCm, CUDA, etc) perhaps a better name could be applied here.

"GPU Settings", perhaps?

**Comment by vladmandic** [comment]
its not even GPU as precision applies to CPU-only. I'll change it to "Compute Settings"
